gui/src/App.tsx

```diff
@@ -152,14 +153,15 @@ function App() {
     };
   }, []);
 
+  const winClass = !/windows/i.test(navigator.userAgent) ? 'win_flags' : '';
```

Comment:
can't we just use the twitter emojis all the time?

Reply:
That's an option too, do people on linux/mac care much about having their native OS flags? I don't know if there's even much of a difference.